BankSDK/GiniBankSDKExample/GiniBankSDKExample/SelectAPIViewController.swift

```diff
@@ -10,44 +10,256 @@ import GiniCaptureSDK
-    @IBAction func launchScreenAPI(_ sender: Any) {
-        delegate?.selectAPI(viewController: self, didSelectApi: .screen)
+    private func startSDK(entryPoint: GiniConfiguration.GiniEntryPoint) {
+        // we should remove keyboard observers to not interfere with others from the SDK
+        removeObservers()
+        // we should hide the keyboard if the SDK is presented
+        dismissKeyboard()
+        delegate?.selectAPI(viewController: self, didSelectEntryPoint: entryPoint)
     }
     
-    @IBAction func launchSettings(_ sender: Any) {
+    // MARK: - User interactions
+
+    @objc func launchSettings(_ sender: UITapGestureRecognizer) {
         delegate?.selectAPI(viewController: self, didTapSettings: ())
     }
     
+    @IBAction func photoPaymentButtonTapped(_ sender: Any) {
+        startSDK(entryPoint: .button)
+    }
+    
+    @objc func ibanCameraIconTapped(_ sender: Any) {
+        startSDK(entryPoint: .field)
+    }
+    
+    // MARK: - Notifications
+    
+    private func addObservers() {
+        NotificationCenter.default.addObserver(self,
+                                               selector: #selector(keyboardWillShow(notification:)),
+                                               name: UIResponder.keyboardWillShowNotification,
+                                               object: nil)
+        
+        NotificationCenter.default.addObserver(self,
+                                               selector: #selector(keyboardWillHide(notification:)),
+                                               name: UIResponder.keyboardWillHideNotification,
+                                               object: nil)
+    }
+    
+    private func removeObservers() {
+        NotificationCenter.default.removeObserver(self, name: UIResponder.keyboardWillShowNotification, object: nil)
+        NotificationCenter.default.removeObserver(self, name: UIResponder.keyboardWillHideNotification, object: nil)
+    }
+    
+    // MARK: - Keyboard handling
+    
+    @objc func keyboardWillShow(notification: Notification) {
+        guard let keyboardFrame = (notification.userInfo?[UIResponder.keyboardFrameEndUserInfoKey] as? CGRect) else { return }
+        
+        if let focusedTextField = focusedFormField, view.frame.origin.y == 0 {
+            
+            let textFieldFrameInParent = view.convert(focusedTextField.frame, from: focusedTextField.superview)
+            let remainingHeight = view.frame.height - textFieldFrameInParent.height - textFieldFrameInParent.origin.y
+            if keyboardFrame.height > remainingHeight {
+                view.frame.origin.y -= keyboardFrame.height - remainingHeight
+                view.layoutIfNeeded()
+            }
+        }
+    }
+    
+    @objc func keyboardWillHide(notification: Notification) {
+        if view.frame.origin.y != 0 {
+            view.frame.origin.y = 0
+            view.layoutIfNeeded()
+        }
+    }
+    
+    private func initializeHideKeyboard() {
```

Comment:
[Suggestion]: the name `initializeHideKeyboard` is a little bit misleading 😅 maybe something into "dismissOnTap"? 🤔